libs/langchain/langchain/llms/bedrock.py

```diff
@@ -404,6 +494,65 @@ def _call(
 
         return self._prepare_input_and_invoke(prompt=prompt, stop=stop, **kwargs)
 
+    async def _astream(
+        self,
+        prompt: str,
+        stop: Optional[List[str]] = None,
+        run_manager: Optional[CallbackManagerForLLMRun] = None,
+        **kwargs: Any,
+    ) -> AsyncGenerator[GenerationChunk, None]:
+        """Call out to Bedrock service with streaming.
+
+        Args:
+            prompt (str): The prompt to pass into the model
+            stop (Optional[List[str]], optional): Stop sequences. These will
+                override any stop sequences in the `model_kwargs` attribute.
+                Defaults to None.
+            run_manager (Optional[CallbackManagerForLLMRun], optional): Callback
+                run managers used to process the output. Defaults to None.
+
+        Yields:
+            AsyncGenerator[GenerationChunk, None]: Generator that asynchronously yields
+            the streamed responses.
+        """
+        async for chunk in self._aprepare_input_and_invoke_stream(
+            prompt=prompt, stop=stop, run_manager=run_manager, **kwargs
+        ):
+            yield chunk
+
+    async def _acall(
+        self,
+        prompt: str,
+        stop: Optional[List[str]] = None,
+        run_manager: Optional[CallbackManagerForLLMRun] = None,
```

Comment:
This should be "run_manager: Optional[AsyncCallbackManagerForLLMRun] = None," to be compatible with the method being overwritten in "BaseLLM".
https://github.com/DLOVRIC2/bedrock-async-support/blob/4fb265c501ab477bd143ed0e6710e2c77341d8d4/libs/langchain/langchain/llms/base.py#L1022